<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZWindow</class>
 <widget class="QMainWindow" name="ZWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 226, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>20</y>
      <width>401</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>13</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 120%;
text-align: center;</string>
     </property>
     <property name="text">
      <string>ПЕРЕМЕЩЕНИЕ ОП ОСИ Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>30</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>201</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>306</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>224</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
border:0px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Режим
движения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
border:0px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Управление</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>223</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_move_step.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="upLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>323</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_up.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="stopLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>323</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_stop.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="downLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>323</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/z_down.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="zstopButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>521</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="dButton">
     <property name="geometry">
      <rect>
       <x>418</x>
       <y>16</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border : 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>35</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 14px;
color: #12222D;
border:0px;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Размер шага</string>
     </property>
    </widget>
    <widget class="QLabel" name="stepLabel">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>25</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/img/selector_50.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>17</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border : 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="aButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border : 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>16</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border : 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/img/z_1.png</pixmap>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>label_8</zorder>
    <zorder>stepLabel</zorder>
    <zorder>bButton</zorder>
    <zorder>cButton</zorder>
    <zorder>aButton</zorder>
    <zorder>dButton</zorder>
   </widget>
   <widget class="QLabel" name="connectionWifi">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/noSignal.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="loadLabel">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>402</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 800;
font-size: 20px;
color: #12222D;
border:0px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>15</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
